jobs/gradle.yml

```diff
@@ -0,0 +1,50 @@
+# gradle
+#
+# Job responsible for building Gradle Wrapper based projects.
+#
+
+parameters:
+  jdkVersions:
+    - name: JDK_8
+      version: "1.8"
+    - name: JDK_11
+      version: "1.11"
+  jobs:
+    - name: Ubuntu_16
+      vmImage: ubuntu-16.04
+    - name: Ubuntu_18
+      vmImage: ubuntu-18.04
+    - name: macOS_10_14
+      vmImage: macos-10.14
+    - name: Windows_2016
+      vmImage: vs2017-win2016
+
+jobs:
+  - ${{ each job in parameters.jobs }}:
+      - job: ${{ job.name }}
+        strategy:
+          matrix:
+            ${{ each jdkVersion in parameters.jdkVersions }}:
+              ${{ jdkVersion.name }}:
+                jdk.version: ${{ jdkVersion.version }}
+        pool:
+          vmImage: ${{ job.vmImage }}
+        steps:
+          - task: Gradle@2
+            inputs:
+              workingDirectory: ""
+              gradleWrapperFile: "gradlew"
+              gradleOptions: "-Xmx3072m"
```

Comment:
where this value comes from? Each project has own needs. Usually what Gradle has by default is enough for small projects. Larger once will adjust this setting in their gradle.properties file. No need for this options to be present here.

Reply:
Makes totally sense. I took it from an example. Will remove it